onnx/backend/test/case/node/topk.py

```diff
@@ -20,7 +20,7 @@ def topk_sorted_implementation(X, k, axis, largest):  # type: ignore
         sorted_values = np.flip(sorted_values, axis=axis)
     topk_sorted_indices = np.take(sorted_indices, np.arange(k), axis=axis)
     topk_sorted_values = np.take(sorted_values, np.arange(k), axis=axis)
-    return topk_sorted_values, topk_sorted_indices
+    return topk_sorted_values, np.array(topk_sorted_indices, dtype=np.int64)
```

Comment:
I thought the test case changes were made in a separate PR...

Reply:
Because this bug only happened in 32-bit environment and I found it in this PR while running 32-bit release CI.